### ISSUE TITLE
Source for editor

### ISSUE THREAD

**Issue by strontian** [oakes/Nightcoders.net#3]
Hey Zach,

Thanks for all your amazing work, it's really cool!

Not really an issue so much as a question - where does the code for the editor that you see in nightcoders.net once you start a project come from? I could find the source for the landing page in the repo, but not the editor itself.

**Comment by oakes** [comment]
The source for the editor is https://github.com/oakes/Nightlight which nightcoders.net pulls in as a dependency

**Comment by strontian** [comment]
Thanks! I'm working on an experimental editor and researching various ways to handle evaluation. The concept of an embedded IDE is quite cool in that regard.

**Comment by oakes** [comment]
Yeah i'm still really enthusiastic about the idea. Most of my work is going towards paravim, which is kind of a successor to nightlight. It definitely makes it easier to eval code since it's in the same process.
